### ISSUE TITLE
From fossil: Filter changes after sleep when launching from shortcut

### ISSUE THREAD

**Issue by mpcjanssen** [mpcjanssen/simpletask-android#61]
Reverting to originally selected filter when phone wakes from sleep. 
(when application opened using a homescreen shortcut)

Steps:
- Create Filter and "Send to Homescreen"
- Close Simpletask.
- Open Simpletask using the newly created shortcut.
- Change to another filter.
- Sleep Phone, then wake. Filter changed back to original filter.

anonymous added on 2014-01-28 01:17:20:
Just noticed that Simpletask returns to original filter not only after waking from sleep, but after adding a new task as well.
**Reporter:** Admin Admin
**Assigned to:** Mark Janssen
**Begin:** 2014-01-30
**Completed:** 0
